content/en/docs/release_notes/3_0_2_20211105_relnotes.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Axway CLI 3.0.2 GA release notes - 5 November 2021
+linkTitle: Axway CLI 3.0.2 GA release notes - 5 November 2021
+description: Review Axway CLI release details such as new features, improvements, and bug fixes.
+weight: 10
+date: 2021-11-05
+---
+
+{{% variables/axway_cli_prod_name %}} 3.0.2 GA - 5 November 2021 is a major release that includes new features, improvements, and bug fixes.
```

Comment:
....is a patch release that includes a new feature and improvements.

Reply:
Fixed